themes/src/main/resources-community/theme/base/account/messages/messages_de.properties

```diff
@@ -339,3 +339,46 @@ access-denied-when-idp-auth=Zugriff verweigert bei Authentifizierung mit {0}
 organizationScopeConsentText=Organisation
 updateReadOnlyAttributesRejectedMessage=Aktualisierung eines schreibgeschützten Attributs abgelehnt
 invalidPasswordNotContainsUsernameMessage=Ungültiges Passwort: Darf den Benutzernamen nicht enthalten.
+locale_ar=عربي
+locale_ja=日本語
+locale_it=Italiano
+error-user-attribute-required=Bitte geben Sie Attribut {0} an.
+locale_ru=Русский
+locale_uk=Українська
+locale_zh-CN=中文简体
+locale_tr=Türkçe
+locale_fi=Suomi
```

Comment:
Hmmm, I thought we were going to get rid of this, should we hold off on this? @ahus1

Reply:
The locales appeared in to message resources, and I set them to empty and not translated only in one of them. I fixed it now in the Weblate UI.